src/pages/docs/deployments/custom-scripts/aws-cli-scripts.md

```diff
@@ -100,17 +100,25 @@ Get-STSCallerIdentity | Select-Object -Property *
 
 Scripts can be run from two different locations: source code added to the step, or a script from a package.
 
-### Source code
+### Inline script
 
-The first option is to add the script source code to the step directly. This is done by selecting the `Source code` option, and entering the source code into the text box in the `Script Content` section.
+The first option is to add the script source code to the step directly. This is done by selecting the `Inline script` option, and entering the source code into the text box in the `Inline Source Code` section.
 
 :::figure
 ![Source Code](/docs/deployments/custom-scripts/images/step-aws-script.png)
 :::
 
+### Git repository
+
+The second option is to run a script from a git repository. This is done by selecting the `Git repository` option, configuring the repository details, and entering the path name of the script file to run.
```

Comment:
Should this read 'entering the path to the script file you want to run'?

Reply:
Fixed